src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/login/SpecialUserLoginPageObject.java

```diff
@@ -123,7 +123,8 @@ public void verifyMessageAboutNewPassword(String userName) {
   }
 
   public void verifyClosedAccountMessage() {
-    waitForElementByElement(messagePlaceholder);
+//    waitForElementByElement(messagePlaceholder);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)